### DISCUSSION TITLE
Dual monitors above two single monitors question

### DISCUSSION THREAD

**Discussion by sigtel** [debauchee/barrier#1966] [Q&A]
This is a bit of a non-issue, more a question on configuration. I run a dual screen Linux above a single Windows and a single MacOS (Server), so the rhs screen on Linux is above the Mac and the lhs screen above the Windows screen. Moving up from Mac is fine, then across the two Linux screens also fine. But I can't move up from Windows to Linux as Barrier sees the only upward path as being from the Mac. Is there a way round that?

**Comment by DeepYogurt** [comment]
Which system is the server and which the clients? Since your Linux system is dual screens, they really are logically one screen I think, correct (ie if each were 2048x1024 then you have one 4096x1024 video layout)? If true then only one of the others can be "adjacent" to this "single" screen I would guess. If you had the two Linux screens each on their own Linux system I would suspect that you could be able to accomplish what you're wanting, but that means another computer in the setup. Good luck...

**Reply by sigtel** [reply]
The top two are dial screens on a single Linux box, so yes logically one screen. Bottom left, under the LHS Linux one is Windows, and bottom right the Mac - and Barrier server. I can see why it happens - exactly as you say - and I can easily live with it. The flexibility Barrier gives me having just one keyboard and mouse (cordless at that) vs occasionally getting the mouse lost is nothing to worry about.
